File: programacion/3_Estructuras_de_Datos/2_Ejercicios_Listas_Tuplas/Ejercicio_3_1_10.py
# Escribir un programa que almacene en una lista los siguientes precios: 50, 75, 46,
# 22, 80, 65, 8 y muestre por pantalla el menor y el mayor de los precios.

precios = [50, 75, 46, 22, 80, 65, 8]

precios_ordenados = sorted(precios)

primero = precios_ordenados[0]
ultimo = precios_ordenados[-1]
print(f"de la lista precios: {precios} , el mayor es: {primero} y el ultimo es: {ultimo}")

File: programacion/4_POO/1_KotlinBasico_EEDD/src/Ejercicio3110.kt
/*Escribir un programa que almacene en una lista los siguientes precios:
50, 75, 46,
22, 80, 65, 8
y muestre por pantalla el menor y el mayor de los precios*/

fun main() {

    val precios = listOf(50, 75, 46, 22, 80, 65, 8)


    val precioMenor = precios.minOrNull()
    val precioMayor = precios.maxOrNull()

    
    println("El precio menor es: $precioMenor")
    println("El precio mayor es: $precioMayor")
}
